src/test/ui/nll/borrowed-universal-error-2.stderr

```diff
@@ -1,17 +1,16 @@
 error[E0597]: `v` does not live long enough
   --> $DIR/borrowed-universal-error-2.rs:16:5
    |
+LL | fn foo<'a>(x: &'a (u32,)) -> &'a u32 {
+   |               ----------     ------- has type `&'a u32`
+   |               |
+   |               has type `&'a (u32,)`
```

Comment:
I'm a bit torn about highlighting `x` here. I mean, it doesn't turn out to *matter* what type `x` has, right? Though maybe that is hard for us to see right now.

Reply:
The intention with highlighting the argument is to show that through the elision rules, it is expected to be the same lifetime as the return type. This isn't useful in this case where the argument and return type have the lifetimes specified.

Reply:
I've added a note that should clarify this intent for the cases where elision rules are relevant and a help for the named lifetime cases.